src/Cli/dotnet/commands/dotnet-tool/restore/ToolRestoreCommandParser.cs

```diff
@@ -1,40 +1,37 @@
 // Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using Microsoft.DotNet.Cli.CommandLine;
+using System.CommandLine;
+using Microsoft.DotNet.Tools.Tool.Common;
 using LocalizableStrings = Microsoft.DotNet.Tools.Tool.Restore.LocalizableStrings;
 
 namespace Microsoft.DotNet.Cli
 {
     internal static class ToolRestoreCommandParser
     {
-        public static Command ToolRestore()
+        public static readonly Option ConfigOption = ToolInstallCommandParser.ConfigOption;
+
+        public static readonly Option AddSourceOption = ToolInstallCommandParser.AddSourceOption;
```

Comment:
Same as global/local. The text between install and restore are actually different. You could try to make the text generic instead.

Please check the convert for tools command for similar issue. And you should remove unused restore/list/uninstall counterpart